src/server.ts

```diff
@@ -28,6 +26,9 @@ export function start(): Promise<ConnectionInfo> {
                         host: 'localhost'
                     });
                 });
+                sspserver.stdout.on('data', data => {
+                    return `${data}`;
```

Comment:
This is even listener, it should just redirect 'data' events to specific channel. Just get 'SSP Server (stdout)' output channel by name the same way it is done [here ](https://github.com/mohitsuman/adapters-vscode/blob/master/src/serverExplorer.ts#L59) and print out 
`data.toString()` into it.
Also add listener fro stderr stream and redirect output to 'SSP Server (stderr)' output channel the same way it suggested for stdout stream.